### PR TITLE
*: fix kvproto dependency

### PR DESCRIPTION
**Pull request by BusyJay**
Use git path instead.

### CODE REVIEW

**Comment by siddontang** [comment]
LGTM

**Comment by siddontang** [comment]
PTAL @hoverbear

File: Cargo.toml [hunk 1]
```diff
@@ -21,8 +21,7 @@ rand = "0.4"
 clippy = {version = "*", optional = true}
 
 [dependencies.kvproto]
-path = "../kvproto"
-#git = "https://github.com/pingcap/kvproto.git"
+git = "https://github.com/pingcap/kvproto.git"
```
Comment (Hoverbear) [inline Cargo.toml]: Since this is a library without a `Cargo.lock` this is very dangerous. It'd be safer to lock it to a particular tag or revision.

**Comment by siddontang** [comment]
Hi @Hoverbear 
Can you give a "LGTM" now?

**Pull request merged**